.travis.yml

```diff
@@ -80,11 +80,11 @@ install:
 
 before_script:
     # Get the test cases
-    - git clone -b develop https://github.com/su2code/TestCases.git ./TestData
+    - git clone -b feature_preconditioning https://github.com/su2code/TestCases.git ./TestData
```

Comment:
I'm sure you intend to change these; just adding a reminder

Reply:
Yep, thanks for the reminder. We've got the classic issue where we need to update both the TestCases repo and code repo simultaneously to avoid failure.. I'll push to TestCases right before we merge this one.